frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1932,17 +1942,20 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                     return false
                 }
 
-                const geoIpPlugin =
-                    pluginsResponse.status === 'fulfilled' &&
-                    pluginsResponse.value.find((plugin) => plugin.url && GEOIP_PLUGIN_URLS.includes(plugin.url))
-                const geoIpPluginId = geoIpPlugin ? geoIpPlugin.id : undefined
+                if (hogFunctionsResponse.status !== 'fulfilled') {
+                    return false
+                }
+
+                const hogFunction = hogFunctionsResponse.value.results.find(
```

Comment:
Might want to filter here for enabled instead as there could be multiple with the first one disabled

Reply:
Oh, good catch: 0644ab022866c6482f1d92a008f84de4b46f3c1c